README.md

```diff
@@ -155,10 +155,34 @@ Blaze-Persistence Spring Data integration dependencies
 ```xml
 <dependency>
     <groupId>com.blazebit</groupId>
-    <artifactId>blaze-persistence-integration-spring-data</artifactId>
+    <artifactId>blaze-persistence-integration-spring-data-api</artifactId>
```

Comment:
The distinction between api and impl module wasn't possible, porbably a leftover of your try. Please revert.